### PR TITLE
add dekalee mailjet bundle configuration

### PR DESCRIPTION
**Pull request by nicolasThal**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: dekalee/mailjet-bundle/0.5/manifest.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "Dekalee\\MailjetBundle\\DekaleeMailjetBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "DEKALEE_MAILJET_API_KEY": "your-api_key",
+        "DEKALEE_MAILJET_SECRET_KEY": "@@your-secret_key",
```
Comment (maxhelias) [inline dekalee/mailjet-bundle/0.5/manifest.json]: Can you by default comment the variables and add the statements in the post-install.txt ?

**Comment by maxhelias** [comment]
Also, think about making interactive rebase or amend. It is much more convenient for the rereading of history Git 😉

**Comment by nicolasThal** [comment]
Thank you @maxhelias for the tips for the post-install.txt file.

I have also squashed all the commits

**Comment by maxhelias** [comment]
It look good 👍

File: dekalee/mailjet-bundle/0.5/manifest.json (hunk 1 above)
Comment (Nyholm) [reply dekalee/mailjet-bundle/0.5/manifest.json]: Why the double at-sign (@)?

Comment (nicolasThal) [reply dekalee/mailjet-bundle/0.5/manifest.json]: @Nyholm I saw it in another recipes.

I will remove it for a better clarity.

**Comment by nicolasThal** [comment]
@Nyholm I have updated the pr and removed the double `@`.

**Comment by Nyholm** [comment]
Did you push your changes?

**Comment by nicolasThal** [comment]
Sorry @Nyholm , I forgot to add the modification when I amended the commit.

Now the modification is pushed :).

**Pull request merged**